src/diffusers/schedulers/scheduling_ddim.py

```diff
@@ -240,6 +249,8 @@ def step(
         # - pred_sample_direction -> "direction pointing to x_t"
         # - pred_prev_sample -> "x_t-1"
 
+        timestep = self.schedule[timestep]
```

Comment:
Here I'm a bit lost - the code now here is more complex than it was before

Reply:
Why are we now passing a different timestep to the scheduler than before that we then revert here?